tests/RestClient/demo.http

```diff
@@ -0,0 +1,137 @@
+@port = 5198
+
+### Note, no descriptor validation yet
+
+### Setup - POST School Year 2025
+POST http://localhost:{{port}}/data/ed-fi/schoolYearTypes
```

Comment:
Friday's demo

Reply:
This will be great for the Project Tanager workgroup meeting on Thursday.